features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/dashdirect/DashDirectViewModel.kt

```diff
@@ -99,7 +99,7 @@ class DashDirectViewModel @Inject constructor(
             .launchIn(viewModelScope)
 
         walletDataProvider
-            .observeBalance()
+            .observeTotalBalance()
```

Comment:
observeBalance -> observeTotalBalance, make it explicity getting the total balance, not necessarily what is spendable.

This balance should be based on the CoinJoin Mode, but that will be handled later.